proto/src/main/proto/Rss.proto

```diff
@@ -174,6 +174,7 @@ message ShuffleRegisterRequest {
   RemoteStorage remoteStorage = 4;
   string user = 5;
   DataDistribution shuffleDataDistribution = 6;
+  int32 maxConcurrencyPerPartitionToWrite = 7;
```

Comment:
Do you need a new data structure like `ShuffleSpecification` to wrap these data structures? Maybe we will add more data structures, we would better to do some classification.

Reply:
I also think this. But it's not necessary, and it will break compatibility

Reply:
Maybe we should introduce `ShuffleSpecification` in the ShuffleTaskInfo first.

Reply:
SGTM. This is more meaningful.